exotic.py

```diff
@@ -407,30 +367,108 @@ def user_input(prompt, type_, val1=None, val2=None):
             return option
 
 
-def check_file_extensions(directory, fileName):
-    # Add / to end of directory if user does not input it
-    if directory[-1] != "/":
-        directory += "/"
```

Comment:
why is this now removed?

Reply:
Not removed, it's lower down. Due to the increase in code, it moved it down on the right-hand side.

Reply:
Ah nevermind- you've tweaked it below.